tests/ui-tests/UITests.swift

```diff
@@ -101,43 +101,6 @@ class UITests: XCTestCase {
         waitForElementToAppear(app.staticTexts["MyUniqueCustomAction"])
     }
     
-    func testShowSystemNotification() {
```

Comment:
Test should still pass right? Why to remove them?

Reply:
It won't work, actually, the deprecated calls were tied into the Storyboard. But it's also not relevant to our SDK anymore.